tests/bootstrap.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+use Doctrine\DBAL\Types\Type;
+
+Type::addType('uuid', 'Doctrine\Tests\Common\DataFixtures\TestTypes\UuidType');
```

Comment:
IS there any particular reason why this is done here and not i.e. in test's `setUp`?

Reply:
It should be done only once per all test cases. It seems that I could move this to `setUpBeforeClass()` but not sure it's better